lib/V8/v8-json.ll

```diff
@@ -26,11 +26,19 @@
 /// @author Copyright 2011, triAGENS GmbH, Cologne, Germany
 ////////////////////////////////////////////////////////////////////////////////
 
+#include <cmath>
+
 #include "Basics/Common.h"
+#include "Basics/operating-system.h"
 #include "Basics/tri-strings.h"
 #include "V8/v8-json.h"
 #include "V8/v8-globals.h"
 
+#ifdef _WIN32
+#include "Basics/win-utils.h"
```

Comment:
Why this?

Reply:
To be honest, I don't remember off the top of my head, but it was to fill a dependency from some macro. We could stand to do a lot of restructuring to better abstract away the OS-dependent implementations, but that's for another day.